### PR TITLE
User-controlled homepage animation

### PR DESCRIPTION
**Pull request by vladmoroz**
https://github.com/radix-ui/website/assets/8441036/322eadc9-9ab9-4b1d-b246-d12d2d8f7e95

### CODE REVIEW

File: pages/themes/build-faster.tsx
Comment (vladmoroz) [inline pages/themes/build-faster.tsx:1]: Don't want to support the old implementation

File: components/Header.tsx [hunk 1]
```diff
@@ -40,6 +40,7 @@ export const Header = ({ children, gitHubLink, ghost }: HeaderProps) => {
       removeEventListener('scroll', handleScroll);
     }
 
+    handleScroll();
```
Comment (vladmoroz) [inline components/Header.tsx:43]: Fixes the bug when switching the homepage slides after you scroll wouldn't show the header in the correct state

File: pages/index.tsx [hunk 2]
```diff
@@ -277,37 +309,43 @@ const codeStyles = {
...
+  '--color-background-image-base': 'var(--sage-1)',
+  '--color-background-image-accent-1': 'var(--teal-a7)',
   '--color-background-image-accent-2': 'var(--mint-7)',
   '--color-background-image-accent-3': 'var(--green-9)',
   '--color-background-image-accent-4': 'var(--sky-5)',
   '--color-background-image-accent-5': 'var(--crimson-3)',
-  '--color-background-image-accent-6': 'var(--mint-6)',
+  '--color-background-image-accent-6': 'var(--mint-a5)',
   '--color-background-image-accent-7': 'var(--teal-5)',
+  // transformOrigin: 'center center',
```
Comment (andy-hook) [inline pages/index.tsx:334]: Redundant?

**Review by andy-hook — APPROVED** [review]

File: pages/index.tsx (hunk 2 above)
Comment (vladmoroz) [reply pages/index.tsx]: Saving these as a comment:
image

I used the homepage a few times to create graphics based on it

**Pull request merged**